code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -167,8 +167,41 @@
 		"is twisting their own neck!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/nerve_pinch
+		)
+
 /datum/species/vulpkanin/handle_death(var/mob/living/carbon/human/H)
 	H.stop_tail_wagging(1)
+	
+/mob/living/carbon/human/proc/nerve_pinch()
+	set category = "Abilities"
+	set name = "Nerve pinch"
+	set desc = "Incapacitate someone who's being completely illogical by pinching a nerve cluster on their neck"
+
+	if(stat)
+		to_chat(src, "You cannot perform the maneuver in this state.")
+		return
+
+	var/list/choices = list()
+	for(var/mob/living/carbon/human/H in view(1,src))
+		var/species = H.get_species()
+		var/race = get_species()
+		if(!H.stat && species == race && src.Adjacent(H) && src != H)
+			choices += H
+
+	var/mob/living/carbon/human/M = input(src,"Who is being too illogical?") in null|choices
```

Comment:
gee you know what this lets me do

this lets me walk next to someone, hit the button, and move the prompt to the side

then I can OPEN PALM SLAM THAT OK BUTTON right as I run by them and get a 5 second knockout stun.